cluster-scope/base/imageregistryconfigs/cluster/config.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: imageregistry.operator.openshift.io/v1
+kind: Config
+metadata:
+  name: cluster
```

Comment:
This looks very different from the one in live: 

```yaml
apiVersion: operator.openshift.io/v1
kind: Config
metadata:
  annotations:
    release.openshift.io/create-only: "true"
  name: cluster
spec:
  managementState: Managed
status: {}
```

Reply:
that's a different `Config` kind, see the API group:
`operator.openshift.io/v1`'s`Config`  vs `imageregistry.operator.openshift.io`'s `Config`.

you'd need to query it as:
```
oc get configs.imageregistry.operator.openshift.io cluster
```

It confused me as well.

Reply:
gotcha thanks!